mypy.ini

```diff
@@ -0,0 +1,82 @@
+[mypy]
+warn_unused_configs = True
+warn_redundant_casts = True
+plugins = pydantic.mypy
+
+# networkx doesn't ship type annotations
+[mypy-networkx.*]
+ignore_missing_imports = True
+
+# Don't check LSST packages generally or even try to import them, since most
+# don't have type annotations.
+
+[mypy-lsst.*]
+ignore_missing_imports = True
+ignore_errors = True
+
+# Do expect to import type annotations from LSST middleware dependencies that
+# should have them, but don't re-check them as they have their own mypy.ini
+# files with their own exceptions.
+
+[mypy-lsst.utils.*]
```

Comment:
How do we know it's importing the types from utils and resources? Also, daf_butler doesn't declare that it is types and doesn't set MYPYPATH in the table file.

Reply:
Ha. I see that's in lsst/daf_butler#647 ...